### ISSUE TITLE
undefined is not an object (evaluating '_react.PropTypes.bool')

### ISSUE THREAD

**Issue by bencodezen** [react-native-training/react-native-elements#564]
### Description

1. Installed `react-native-elements`
2. Imported it into my screen with `import { SideMenu, List, ListItem } from 'react-native-elements'`
3. Got this error:

https://cl.ly/1B0m1v0m0k3g

Also please include the version of `react-native-elements` that you are using.

### Reproduction Steps and Sample Code

Here is my `package.json`

```json
{
  "name": "metro-app",
  "version": "0.0.0",
  "description": "Hello Expo!",
  "author": null,
  "private": true,
  "main": "node_modules/expo/AppEntry.js",
  "dependencies": {
    "axios": "^0.16.2",
    "expo": "^19.0.0",
    "react": "^16.0.0-alpha.12",
    "react-native": "https://github.com/expo/react-native/archive/sdk-19.0.0.tar.gz",
    "react-native-device-info": "^0.10.2",
    "react-native-elements": "^0.16.0",
    "react-native-htmlview": "^0.12.0",
    "react-native-menu": "^0.21.0",
    "react-navigation": "^1.0.0-beta.12",
    "react-redux": "^5.0.5",
    "redux": "^3.7.2",
    "redux-persist": "^4.8.3",
    "redux-thunk": "^2.2.0"
  },
  "devDependencies": {
    "reactotron-react-native": "^1.12.1",
    "remote-redux-devtools": "^0.5.12"
  }
}
```

### Solution

My guess is the propType separation from React is causing the issue?

**Comment by iRoachie** [comment]
Hey @bencodezen This will be resolved in the next version 0.17. The library that it's coming from is react-native-side-menu. The fix has already been made in their library https://github.com/happypancake/react-native-tab-navigator/pull/152, just a new version hasn't been published. For the meantime, you can use our `next` branch, which has the fix.

As soon as a new version is published, we'll switch to it.

**Comment by mgenov** [comment]
The release is available: 0.3.4 is the latest of 24 releases

**Comment by iRoachie** [comment]
Already bumped it @mgenov https://github.com/react-native-training/react-native-elements/pull/551/commits/6db4be5fab7fa23171d8b7f17dc3dd9e686d6d19

**Comment by mgenov** [comment]
Got it working on my project with that revision. Thanks @iRoachie

**Comment by ricardoalcantara** [comment]
@mgenov I still got this error using the latest react-native-elements package (0.16) published at NPM.

**Comment by iRoachie** [comment]
@ricardoalcantara Use the `next` branch for now. It's not published as yet. `0.17` still in the works

**Comment by ricardoalcantara** [comment]
I did have problem to point the git link+branch, maybe because I am behind a proxy I don't know. I will stick with the 0.16 and keep using react 16.0.0-alpha.12 until the 0.17 is published. Thanks.

**Comment by iRoachie** [comment]
This is fixed in 0.17, which is now live
